Extensions/AzureExp/Src/Tasks/ManageExperimentV1/task.json

```diff
@@ -16,12 +16,12 @@
         "Patch": 0
     },
```

Comment:
Can you confirm that the taskId is a new guid and not the Id used in the POC?

Reply:
Added new task guid